README.md

```diff
@@ -21,6 +21,8 @@
          <p style="color:purple;font-size:18px;">"I was born in the Philippines and moved to Txas at 6 years old. I've also visited other countries like Japan, Mexico and the states Californiaaaaa, Florida, Nevada, South Carolinaaaaa.</p>
       </body>
 </html>
+![Ring_08-04-2021-19-38-47](https://user-images.githubusercontent.com/63130172/133013497-bdee36d4-7d3b-42c6-a6ff-6be7358e4207.JPG)
```

Comment:
I think the conflict is something to do with this picture? It is different than the one in embedded now.